### PR TITLE
Epmlstrtai 44

### PR DESCRIPTION
**Pull request by Walrus94**

### CODE REVIEW

File: Java/JDI/jdi-cucumber-test-generator/src/main/webapp/WEB-INF/static/views/newSuits.html [hunk 1]
```diff
@@ -127,6 +127,83 @@
 
                                 <div class="cucumber-test-editor-container" >
                                     <div id="steps_container"></div>
+                                    <div id="steps_container">
```
Comment (FunCat) [inline Java/JDI/jdi-cucumber-test-generator/src/main/webapp/WEB-INF/static/views/newSuits.html]: Please, delete all added lines. It should look like this:
\<div class="cucumber-test-editor-container" >
    \<div id="steps_container"></div>
\</div>

**Review by FunCat — APPROVED** [review]

File: Java/JDI/jdi-cucumber-test-generator/src/main/java/com/epam/test_generator/services/SuitService.java [hunk 2]
```diff
@@ -58,4 +62,75 @@ public SuitDTO addSuit(SuitDTO suitDTO) {
         mapper.map(suit, suitDTO);
         return suitDTO;
     }
+
+    public String generateStream(Long suitId, List<Long> CaseIds) throws IOException {
+        Suit suit = suitDAO.findOne(suitId);
+
```
Comment (pomkine) [inline Java/JDI/jdi-cucumber-test-generator/src/main/java/com/epam/test_generator/services/SuitService.java]: Please remove commented code

File: Java/JDI/jdi-cucumber-test-generator/src/main/java/com/epam/test_generator/services/SuitService.java [hunk 3]
```diff
@@ -58,4 +62,75 @@ public SuitDTO addSuit(SuitDTO suitDTO) {
...
+//        List<Step> stepList2 = new ArrayList<>();
+//        stepList2.add(step3);
+//        stepList2.add(step4);
+//        caze1.setSteps(stepList2);
+//
+//        caseList.add(caze2);
+//
+//        suit.setCases(caseList);
+
+        StringBuilder output = new StringBuilder("Feature: ");
```
Comment (pomkine) [inline Java/JDI/jdi-cucumber-test-generator/src/main/java/com/epam/test_generator/services/SuitService.java]: Let's use a template here. [Freemarker](http://freemarker.org/) is a pretty handy template processing lib.

File: Java/JDI/jdi-cucumber-test-generator/src/main/java/com/epam/test_generator/controllers/SuitController.java [hunk 4]
```diff
@@ -78,6 +83,18 @@ public String getMainPage() {
         return new ResponseEntity<>(suitDTO, HttpStatus.UNPROCESSABLE_ENTITY);
     }
 
+    @RequestMapping(value = "/downloadFeatureFile", method = RequestMethod.GET)
```
Comment (pomkine) [inline Java/JDI/jdi-cucumber-test-generator/src/main/java/com/epam/test_generator/controllers/SuitController.java]: why do we need caseIds here?

**Review by roma0297 — APPROVED** [review]

**Pull request merged**